_posts/2018-01-21-ITE-the-scheduler.md

```diff
@@ -0,0 +1,51 @@
+---
+title: In the Engine - The Scheduler
+description: Stuff
+author: dedmen
+layout: default
+category: ITE
+---
+
+Now dedmen just needs to get some good writing skills.
+
+
+When you [spawn](https://community.bistudio.com/wiki/spawn), [execVM](https://community.bistudio.com/wiki/execVM), [exec](https://community.bistudio.com/wiki/execVM) or [execFSM](https://community.bistudio.com/wiki/execFSM) a script it is added to an internal list of scheduled scripts.
+
+
+There is one scheduler cycle per frame (`siScr` in Engine Profiler (New article to be linked here after it's written)) which happens right after the EachFrame event handlers. (Maybe future article about that)
+
+At the beginning of a scheduler cycle a Timer is started, it will be used to keep track of the scheduler time limit.
```

Comment:
> Timer 
Random capitalization?

Reply:
Ah I think I commented about that too.
Yeah https://github.com/arma3/arma3.github.io/pull/2#discussion_r169875788